src/lyapunovspectrum.jl

```diff
@@ -131,44 +130,76 @@ function liap_jetcoeffs!(eqsdiff!, t::Taylor1{T}, x::Vector{Taylor1{U}},
     nothing
 end
 
-"""
-    liap_taylorstep!(f, x, dx, xaux, δx, dδx, jac, t0, t1, x0, order, abstol, vT)
-
-Similar to [`taylorstep!`](@ref) for the calculation of the Liapunov
-spectrum. `jac` is the linearization of the equations of motion,
-and `xaux`, `δx`, `dδx` and `vT` are auxiliary vectors.
+function variational_eqs!(t::Taylor1{T}, x::Vector{Taylor1{U}},
+        dx::Vector{Taylor1{U}}, jac::Array{Taylor1{U},2}, eqsdiff_jac! =Nothing) where {T<:Real, U<:Number}
+    # Dimensions of phase-space: dof
+    nx = length(x)
+    dof = size(jac, 1)
+    if eqsdiff_jac! != Nothing
+        # Stability matrix
+        eqsdiff_jac!(jac, t, x)
```

Comment:
I most admit that I dislike the name `eqsdiff_jac!`; it reminds me to be some sort of differential equation related to the jacobian (which are the variationals!), but refers to the user provided calculation of the Jacobian. I think that `userjacobian!`, `jacobianfunc!` or something like that (with or without `_`) is descriptive enough to be a good name.